streams/src/main/java/org/apache/kafka/streams/kstream/internals/MaterializedInternal.java

```diff
@@ -25,8 +25,16 @@
 
 public class MaterializedInternal<K, V, S extends StateStore> extends Materialized<K, V, S> {
```

Comment:
can be package private I think -- also constructors and all other methods.

Reply:
no it can't be as it is used from `o.a.k.streams`